### PR TITLE
First version of "Users" is ready

### PR DESCRIPTION
**Pull request by alix-tz**
the page still needs a good proofreading
and a few illustrations
and more information on a couple of featues

### CODE REVIEW

**Comment by FloChiff** [comment]
For the proofreading, here are the typos I have noticed in the documentation : 
- l. 32 --> it should be "corner" and not "cornier"
- l. 52 --> a "they" is missing in the phrase, I think it should be "that _they_ have the right"
- l. 64 --> "detailed" should have only one 'l' and "task" at the end of the sentence should not have any 's' as it is "a task"
- l. 88 --> it should be "the main purpose" and not "they main purpose"
- l. 95 --> it should be "member" and not "memer"
- l. 97 --> "transferring" should be written with two consecutives 'r' and not juste one

**Comment by HugoSchtr** [comment]
I can make a few illustrations asap!

**Review by HugoSchtr — COMMENTED** [review]
It looks good! I juste have one question regarding line 100. Should we refer to the version of eScriptorium we're using for writing the documentation?

**Review by FloChiff — APPROVED** [review]

File: docs/walkthrough_users.md [hunk 1]
```diff
@@ -9,4 +9,95 @@ date: 2023-01-31
...
+
+- Creating a team,  
+- Leaving a team,  
+- Adding or removing members from a team,  
+- Transfering ownership to another memer.  
+
+Only the owner of a team is allowed to add or remove members or to transfer ownership. The owner of a team cannot leave it without first transfering the ownership to another user.
+
+??? Note "No deleting, no renaming"
+    It is currently impossible to delete a team or to rename a team.
```
[the remark this answers is not in the document]
Comment (alix-tz) [reply docs/walkthrough_users.md]: We can add a software version for the whole tutorial, I'll try to fix it there

**Comment by alix-tz** [comment]
I think this branch is ready, Floriane or Hugo, do you want to squash and merge it?

File: docs/walkthrough_users.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (HugoSchtr) [reply docs/walkthrough_users.md]: nice! :)

**Comment by HugoSchtr** [comment]
Merging in three...two...one...

**Pull request merged**